src/nrnmpi/nrnmpi_dynam.cpp

```diff
@@ -95,9 +95,30 @@ char* nrnmpi_load(int is_python) {
     pmes = static_cast<char*>(malloc(4096));
     assert(pmes);
     pmes[0] = '\0';
+
+    // If libmpi already in memory, find name and dlopen that.
+    void* sym = dlsym(RTLD_DEFAULT, "MPI_Initialized");
+    if (sym) {
```

Comment:
@nrnhines : windows CI failing with:

```
D:/a/nrn/nrn/src/nrnmpi/nrnmpi_dynam.cpp: In function 'char* nrnmpi_load(int)':
[1146](https://github.com/neuronsimulator/nrn/runs/6008133076?check_suite_focus=true#step:6:1146)
D:/a/nrn/nrn/src/nrnmpi/nrnmpi_dynam.cpp:100:23: error: 'RTLD_DEFAULT' was not declared in this scope
[1147](https://github.com/neuronsimulator/nrn/runs/6008133076?check_suite_focus=true#step:6:1147)
  100 |     void* sym = dlsym(RTLD_DEFAULT, "MPI_Initialized");
[1148](https://github.com/neuronsimulator/nrn/runs/6008133076?check_suite_focus=true#step:6:1148)
      |                       ^~~~~~~~~~~~
[1149](https://github.com/neuronsimulator/nrn/runs/6008133076?check_suite_focus=true#step:6:1149)
D:/a/nrn/nrn/src/nrnmpi/nrnmpi_dynam.cpp:102:9: error: 'Dl_info' was not declared in this scope
[1150](https://github.com/neuronsimulator/nrn/runs/6008133076?check_suite_focus=true#step:6:1150)
  102 |         Dl_info info;
```

Reply:
I'm planning on trying out the dlfcn.c and dlfcn.h files from https://github.com/dlfcn-win32/dlfcn-win32 . After tomorrow morning I won't have access to a windows VBox guest til next Thursday.

Reply:
The above works on Windows.

Reply:
I have a question about how to neatly incorporate dlfcn.h and dlfcn.c for MINGW builds. At one extreme, the above dlfcn-win32 repository can be dealt with as a submodule. At the other extreme, dlfcn.h and dlfcn.c can be copied to nrn/src/mswin and be a part of the nrn repository. My preference is the latter but I'm happy to entertain objections to that. I wouldn't be surprised to find a future msys upgrade to have these files as part of the toolchain, in which case our private copy would be discarded.